cmd/ebitenmobile/gomobile.go

```diff
@@ -30,6 +30,15 @@ import (
 //go:embed gobind.go
 var gobind_go []byte
 
+//go:embed bindings/EbitenViewController.m
```

Comment:
`bindings` sounds a little odd since this includes `EbitenViewController.h`

Rename the directory to `_files` (startings with `_`) so that the Go compiler will ignore it.